webpack_config/config.js

```diff
@@ -4,7 +4,9 @@ const path = require('path');
 module.exports = {
   port: process.env.HTTPS ? 3443 : 3000,
   title: 'MEW',
-  publicPath: process.env.BUILD_GH_PAGES ? '/react-semantic.ui-starter/' : '/',
+  publicPath: process.env.BUILD_GH_PAGES
+    ? '/react-semantic.ui-starter/'
+    : process.env.NODE_ENV === 'production' ? './' : '/',
```

Comment:
Would rather key in on a `DOWNLOADABLE` env variable. I think the idea here is that producing the downloadable version should be considered special, not the default.